person.rb

```diff
@@ -1,7 +1,8 @@
+require_relative 'rental'
 require_relative 'nameable'
 
 class Person < Nameable
-  attr_reader :id
+  attr_reader :id, :rentals
```

Comment:
### Your Attr
```
  attr_reader :id, :rentals
  attr_accessor :name, :age
```
### Expected Attr from Previous Reviewer

```
    attr_reader :id
    attr_accessor :name, :age, :rentals
```

- [ ] Kindly make your attr as suggested by the previous reviewer :blush: